bootstrap.sh

```diff
@@ -310,6 +310,16 @@ software_update() {
   fi
 }
 
+# Let users know that there is system updates to install
+software_update_reminder() {
+  logn "Checking for software updates:"
+  if softwareupdate -l 2>&1 | grep $Q "No new software available."; then
+    logk
+  else
+    log "There are software updates for your system to install. Please check for updates."
+  fi
+}
```

Comment:
Should this be merged into `software_update` fn that accepts a param? (`software_update` is still being called here too)